cmd/snap/cmd_snap_op.go

```diff
@@ -0,0 +1,124 @@
+// -*- Mode: Go; indent-tabs-mode: t -*-
+
+/*
+ * Copyright (C) 2016 Canonical Ltd
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License version 3 as
+ * published by the Free Software Foundation.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ */
+
+package main
+
+import (
+	"time"
+
+	"github.com/ubuntu-core/snappy/client"
+	"github.com/ubuntu-core/snappy/i18n"
+)
+
+func wait(client *client.Client, uuid string) error {
+	for {
+		op, err := client.Operation(uuid)
+		if err != nil {
+			return err
+		}
+
+		if !op.Running() {
+			return op.Err()
+		}
+
+		time.Sleep(100 * time.Millisecond)
+	}
+}
+
+var (
+	shortAddHelp        = i18n.G("Add a snap to the system")
+	shortRemoveHelp     = i18n.G("Remove a snap from the system")
+	shortPurgeHelp      = i18n.G("Purge a snap's data from the system")
+	shortRefreshHelp    = i18n.G("Refresh a snap in the system")
+	shortRollbackHelp   = i18n.G("Rollback a snap to its previous known-good version")
+	shortActivateHelp   = i18n.G("Activate a snap that is installed but inactive")
+	shortDeactivateHelp = i18n.G("Deactivate an installed active snap")
+)
+
+var longAddHelp = i18n.G(`
+The add command installs and activates the named snap in the system.
+`)
+
+var longRemoveHelp = i18n.G(`
+The remove command removes the named snap from the system.
+
+The snap's data is currently not removed; use purge for that. This behaviour
+will change before 16.04 is final.
+`)
+
+var longPurgeHelp = i18n.G(`
+The purge command removes the data of the named snap from the system.
+`)
+
+var longRefreshHelp = i18n.G(`
+The refresh command refreshes (updates) the named snap.
+`)
+
+var longRollbackHelp = i18n.G(`
+The rollback command reverts an installed snap to its previous revision.
+`)
+
+var longActivateHelp = i18n.G(`
+The activate command activates an installed but inactive snap.
+
+Non-active snaps' apps are not available for use.
```

Comment:
Non-active sounds a bit weird to me (but feel free to leave it as is). How about:

```Snaps that are not active don't have their apps available for use.```